### PR TITLE
 2019 update - Energy section extended to include description of policy implementation

### PR DESCRIPTION
**Pull request by OFR-IIASA**

### CODE REVIEW

File: source/energy/policy.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Representation of single country Nationally Determined Contributions (NDCs)
+---------------------------------------------------------------------------
+The targets formulated in the NDCs come in many different flavors. This applies to the sectors and gasses, which policies cover, but also how these are expressed and quantified. In the global energy model, four broad categories of policy types related to the NDCs are represented, each of which is translated into a set of constraints.
+   1. Emission targets
+   2. Energy shares
+   3. Capacity or generation targets
+   4. Macro-economic targets
+A detailed description of the methodological implementation of the NDCs in the global energy model, along with an extensive list of the energy related targets considered can be found in Rogelj et al. (2017) :cite:`rogelj_indc_2017`. Additional policies implemented in the model can also be found in ('what reference for the CD_Links related policies?`)
```
Comment (fabiosferra) [inline source/energy/policy.rst]: Please check CD_Links reference

File: source/energy/policy.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Emission targets
+----------------
+Country specific emission reduction targets are specified either in relation to a reference emission trajectory (in the form of a baseline or business as usual scenario (BAU)). The targets themselves are expressed as either (1) absolute reduction, (2) a percentage reduction or (3) intensity reductions e.g. emissions per GDP or per capita. In order to account for these different reduction targets in the global energy model, the targets are translated so that a regionally specific upper bound on emissions can be formulated. If not further specified, emission constraints are assumed to apply to all sectors and all gases, i.e. total GHGs.
```
Comment (fabiosferra) [inline source/energy/policy.rst]: Please check first sentence: targets are specified either in relation to a reference emission trajectory (...), or [a base year reference?]

File: source/energy/policy.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Separately biofuel shares are implemented specifically for the transport sector.
+
+
+Capacity and generation targets
+-------------------------------
+Some NDCs specify capacity installation targets, e.g. for planned power plants which will be operational by a certain year.  Others specify that a given energy commodity will come from a specific source, for example a certain amount of electricity will stem form a specific intermittent renewable source or nuclear. These targets types are implemented in the model as lower bounds on generation.
+
+
+Macro-economic targets
```
Comment (fabiosferra) [inline source/energy/policy.rst]: The description seems empty here.

Comment (OFR-IIASA) [reply source/energy/policy.rst]: @volker-krey maybe you can provide input on this section, explaining how some of the policies were implemented by adjusting macro related parameters?

File: source/energy/policy.rst (hunk 1 above)
Comment (OFR-IIASA) [reply source/energy/policy.rst]: @volker-krey Marks paper is still not published. Do you have a suggestion for an alternative source to cite the currently implemented policies?

File: source/energy/policy.rst (hunk 2 above)
Comment (OFR-IIASA) [reply source/energy/policy.rst]: @fabiosferra - updates made, please check if this is now clear?

File: source/energy/policy.rst (hunk 1 above)
Comment (jkikstra) [reply source/energy/policy.rst]: @OFR-IIASA would this also work, or are you referring to other, later policy implementations?
Rogelj, J., Fricko, O., Meinshausen, M., Krey, V., Zilliacus, J.J. and Riahi, K., 2017. Understanding the origin of Paris Agreement emission uncertainties. Nature Communications, 8, p.15748.
https://www.nature.com/articles/ncomms15748

File: source/energy/policy.rst (hunk 3 above)
Comment (jkikstra) [reply source/energy/policy.rst]: @volker-krey a gentle reminder here.

File: source/energy/policy.rst (hunk 1 above)
Comment (OFR-IIASA) [reply source/energy/policy.rst]: @jkikstra the suggested publication is already part of the description.  :cite:`rogelj_indc_2017`. This describes the methodology.
The second publication is a reference we require for "which" policies we have implemented. This was supposed to be part of a paper published by a colleague at PBL. The paper is still not out, so I was wondering if there is another source we can quote.  @volker-krey Possibly we can link to http://www.climatepolicydatabase.org/

**Comment by khaeru** [comment]
Since this PR has seen no activity in 5 months, I am going to merge it so that we can complete the “2019” update of the docs. I created two follow-up issues—#42 and #43—and made one commit adding references to these, inline.

**Pull request merged**